src/metabase/driver/crate/generic_sql.clj

```diff
@@ -0,0 +1,37 @@
+(ns metabase.driver.crate.generic-sql
+  (:require [metabase.driver.generic-sql :as sql]
+            [korma.core :as k]
```

Comment:
Do you mind putting the `korma` require above the Metabase ones? We keep them ordered like:

1.  Clojure namespaces
2.  3rd-party namespaces
3.  Metabase namespaces